Sources/Automerge/path.swift

```diff
@@ -4,6 +4,7 @@ import enum AutomergeUniffi.Prop
 typealias FfiPathElem = AutomergeUniffi.PathElement
 typealias FfiProp = AutomergeUniffi.Prop
 
+/// The path to a specific object identifier within an Automerge document.
```

Comment:
I think rather than saying it's a path to an object identifier it might be better to say a "path to a property within an object"?